css/fonts.css

```diff
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: "GothamPro";
+  src:
+    url("/fonts/GothaProReg.woff") format("woff"),
+    url("/fonts/GothaProReg.woff") format("woff2");
+  font-weight: normal;
+}
+    
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/chudik_css_end/pullRequest?prid=6014586)